src/evict/evict_lru.c

```diff
@@ -2533,7 +2538,8 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
             if (cache->evict_aggressive_score > 0)
                 --cache->evict_aggressive_score;
             WT_STAT_CONN_INCR(session, cache_timed_out_ops);
-            __wt_verbose_notice(session, WT_VERB_TRANSACTION, "%s", session->txn->rollback_reason);
+            __wt_verbose_notice(session, WT_VERB_TRANSACTION, "eviction worker block reason: %s",
```

Comment:
Ditto - https://github.com/wiredtiger/wiredtiger/pull/10062/files#r1439114178

Reply:
hi, @Siddhartha8899 
The reason for adding log information here is that I cannot quickly locate where the code was printed from the log. Adding this information can help us quickly locate the corresponding code, making it easier for beginners to learn.
I change the code as following:
 __wt_verbose_notice(session, WT_VERB_TRANSACTION, "worker block reason: %s",

What's your opinion？
thanks.

Reply:
Makes sense, in that case, I would suggest - 
`__wt_verbose_notice(session, WT_VERB_TRANSACTION, "rollback reason: %s", ...`